go.mod

```diff
@@ -6,48 +6,48 @@ require (
 	github.com/Jeffail/gabs v1.4.0
 	github.com/go-logr/logr v0.4.0
 	github.com/google/uuid v1.1.2
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.2
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.14.0
 	github.com/pavel-v-chernykh/keystore-go v2.1.0+incompatible
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.6.1
+	github.com/stretchr/testify v1.7.0
 	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/code-generator v0.20.2
+	k8s.io/api v0.21.3
```

Comment:
Out of curiosity how did you come up with these version numbers? Here for example I see that v0.21.4 could be used (published on August 12th).
And in fact, any version specified here will end up replaced with v0.21.2 because of the replace directive below. So why not use v0.21.2?

Reply:
Yeah, we can use 0.21.2 directly or update to newer one. Those versions come from whatever was created by the operator-sdk in it's go.mod example.